### PR TITLE
Replace rao range action apply by core action modification

### PR DESCRIPTION
**Pull request by pjeanmarie**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [ ] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

**Comment by pjeanmarie** [comment]
Do not merge: Need next core version (6.4.0), then pom need to be updated (6.4.0-SNAPSHOT -> 6.4.0) and changes on CI files need to be removed.

File: data/crac/crac-impl/src/main/java/com/powsybl/openrao/data/cracimpl/PstRangeActionImpl.java [hunk 1]
```diff
@@ -108,9 +109,16 @@ public double getMaxAdmissibleSetpoint(double previousInstantSetPoint) {
 
     @Override
     public void apply(Network network, double targetAngle) {
-        PhaseTapChanger phaseTapChanger = getPhaseTapChanger(network);
+        getPhaseTapChanger(network);
```
Comment (pet-mit) [inline data/crac/crac-impl/src/main/java/com/powsybl/openrao/data/cracimpl/PstRangeActionImpl.java:112]: Do you think it is necessary to keep this line, or can we rely on powybl to throw an exception it the pst doesn't exist? (knowing that, in most cases, the Crac is filtered by cross checking it with the network, during import)

Comment (pjeanmarie) [reply data/crac/crac-impl/src/main/java/com/powsybl/openrao/data/cracimpl/PstRangeActionImpl.java:112]: Yes powsybl can raise exception but I need to change the apply I used: .apply(network) -> .apply(network, true, ReportNode.NO_OP)
I think I need to use this apply for other range actions too, is that correct?

Comment (pet-mit) [reply data/crac/crac-impl/src/main/java/com/powsybl/openrao/data/cracimpl/PstRangeActionImpl.java:112]: yes it is safer

Comment (pjeanmarie) [reply data/crac/crac-impl/src/main/java/com/powsybl/openrao/data/cracimpl/PstRangeActionImpl.java]: done

**Comment by pet-mit** [comment]
@pjeanmarie please fix the DCO issue

File: data/crac/crac-impl/src/test/java/com/powsybl/openrao/data/cracimpl/PstRangeActionImplTest.java [hunk 2]
```diff
@@ -87,16 +88,16 @@ void applyOutOfBound() {
...
+        assertEquals("No matching transformer found with ID:unknownNetworkElement", exception.getMessage());
     }
 
     @Test
     void applyOnTransformerWithNoPhaseShifter() {
         Network network = Network.read("TestCase12Nodes_no_pst.uct", getClass().getResourceAsStream("/TestCase12Nodes_no_pst.uct"));
         PstRangeAction pstRa = pstRangeActionAdder.add();
-        OpenRaoException exception = assertThrows(OpenRaoException.class, () -> pstRa.apply(network, 50));
-        assertEquals("Transformer BBE2AA1  BBE3AA1  1 is not a PST but is defined as a TapRange", exception.getMessage());
+        PowsyblException exception = assertThrows(PowsyblException.class, () -> pstRa.apply(network, 5));
```
Comment (obrix) [inline data/crac/crac-impl/src/test/java/com/powsybl/openrao/data/cracimpl/PstRangeActionImplTest.java:99]: 50 -> 5, is there any reason for this change ?

**Review by obrix — APPROVED** [review]
Good for me, just a small comment.

**Comment by pjeanmarie** [comment]
I think there is no correction needed from my side but correct me if I am wrong.

**Review by pet-mit — APPROVED** [review]

**Pull request merged**